apps/core-app/src/main/modules/database/index.test.ts

```
@@ -0,0 +1,123 @@
+
+vi.mock('../../../../resources/db/locator.json?commonjs-external&asset', () => ({
+  default: '/tmp/resources/db/locator.json'
+}))
+
+vi.mock('../box-tool/addon/files/file-provider', () => ({
+  fileProvider: {
+    isSearchIndexWorkerBusy: fileProviderBusyMock
+  }
+}))
+
+import { dbWriteScheduler } from '../../db/db-write-scheduler'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)